server/src/main/java/org/apache/druid/server/security/AuthenticationUtils.java

```diff
@@ -27,6 +27,16 @@
 
 public class AuthenticationUtils
 {
+  public static void addAllowHttpMethodsFilter(ServletContextHandler root, List<String> allowedHttpMethods)
```

Comment:
This functionality isn't related to authentication. It would more properly belong in JettyServerInitUtils.

Reply:
Done